src/main/resources/assets/paths/draftAttachments.yaml

```
@@ -0,0 +1,61 @@
+get:
+  summary: Get a specific file attachment associated with a draft
+  description: Provides the an individual file attachment connected to a draft.
+  tags:
+    - Draft
+  parameters:
+    - name: draftUUID
+      in: path
+      description: The UUID identifying the draft
+      required: true
+      schema:
+        type: string
+        format: UUID v4.
+    - name: fileId
+      in: path
+      description: The integer identifying the attachment.
+      required: true
+      schema:
+        type: integer
+  responses:
+    200:
+      description: File contents.
+      content:
+        application/octet-stream:
+          schema:
+            type: string
+            format: binary
+    401:
+      description: Unauthorized.  Did you remember to authenticate?
```

```suggestion
      description: User not authorized to get file attachment
```

I would suggest switching this language to more neutral language.